manual/Manual.md

```diff
@@ -50,6 +53,9 @@ To destroy a cable, click on the input port at the end of the
 connection. Again, the "Undo" button can be used to bring back
 any previously destroyed cables.
 
+### Modulation Ports
+Purple modulation ports are available on all modulation processors. When connected, these ports allow for one processors modulation signal to be passed to another. A processor with a connected modulation input will use that input instead of it's own Rate parameter. This allows multiple modulation processors to be synced to the same rate, creating unique effects possibilities.
```

Comment:
```suggestion
Purple modulation ports are available on all modulation processors. When connected, these ports allow for one processor's modulation signal to be passed to another. A processor with a connected modulation input will use the modulation signal from that input instead of generating its own. This feature allows multiple modulation processors to be synced to the same modulation signal, creating unique effects possibilities.
```